### DISCUSSION TITLE
Peer-feedback question for review protocol

### DISCUSSION THREAD

**Discussion by AnneliesPieterman-Bos** [asreview/asreview#2237] [Q&A]
Hi all,

I am soon going to preregister my systematic review protocol, for which I am going to use the SAFE-procedure in ASReview. Before the actual preregistration, I am looking for 'critical friends' to review my protocol and provide me with feedback. That way, I hope to prevent some 'mistakes' that will otherwise maybe only be noted by peer-reviewers of the final manuscript. So, please review my protocol! You can find it in the document attached and on OSF: [250527_Preregistration_review_protocol_v0.1.pdf](https://github.com/user-attachments/files/20477868/250527_Preregistration_review_protocol_v0.1.pdf), [osf.io/9n56h](https://osf.io/9n56h) 

My screening process is now planned as followed:

The selection process will consist of two major phases:
- 	Title-abstract screening, supported by the AI-tool ASReview
- 	Full text screening

### Title-abstract screening
We will use the most recent version of the software ASReview (version 2.0.2 or later) (ASReview LAB developers, 2025). We will follow the SAFE procedure (Boetje & van de Schoot, 2024) as follows:

**Phase 1 – Screen a random set for training data**
- Dataset: all (deduplicated) articles identified through the search strategy
- Stratify the data according to “Science & Society” subtopic (Table 1) in a reference manager
- Manual method: Screen each stratum in a reference manager (to form a training dataset for phase 2)
- Screener(s): screener 1
- Stopping rules for each stratum: 
-- A minimum of 1% of total records has been screened
-- At least 1 relevant record has been identified
- Determine the estimate of relevant records in the full dataset (RR_T) with the following formula:
	RR_T = ((RR_t)/t)×T
	Where t is the total number of records screened, RR_t the total number of relevant records identified when reaching the stopping rules (of all strata combined), and T the total number of records in the dataset.

**Phase 2 – Apply active learning**
- 	Dataset: all (deduplicated) articles identified through the search strategy
- 	ASReview model: Naive Bayes classifier and TF-IDF feature extractor (computationally cheap and efficient)
- 	Prior knowledge: Train the model using the dataset from phase 1 as prior knowledge
- 	Screener(s): screener 1 and 2
- 	During screening, visually assess the recall plot until a plateau has been visually identified, if so, check stopping rules
- 	Stopping rules:	
--	All previously identified key papers are marked as relevant
-- 	At least twice the estimate of relevant records (RR_T) have been screened. 
-- 	A minimum of 10% of total dataset has been screened
-- 	No relevant records have been identified in the last 50 records

**Phase 3 – Find more relevant records with a different model**
- 	Dataset: all (deduplicated) articles identified through the search strategy
- 	ASReview model: neural network model classifier and sBert feature extractor (deep learning)
- 	Prior knowledge: Train the model using labeled data from phase 2 as prior knowledge 
- 	Screener(s): screener 1 and 2
- 	Stopping rule:
-- 	No relevant records have been identified in the last 50 records

**Phase 4 – Evaluate quality**
- 	Dataset: all records labeled as irrelevant
- 	ASReview model: naive Bayes classifier and TF-IDF feature extractor
- 	Train the model using the 10 highest- and lowest-ranked records from phase 2 as prior knowledge
- 	Screener(s): screener 3
- 	Stopping rule:
-- 	No relevant records have been identified in the last 50 records
- 	Additional quality checks (screener 1):
-- 	Use forward and backward citations with the articles identified as relevant to check for missing articles in the final inclusions

### Full text screening
- 	Dataset: all records labeled as relevant after the title-abstract screening phase
- 	The complete author team screens all records identified as relevant to exclude irrelevant articles based on inclusion criteria

ASReview LAB developers. (2025). ASReview LAB - A tool for AI-assisted systematic reviews (2.0.2). Zenodo. https://doi.org/10.5281/zenodo.14999141

Boetje, J., & van de Schoot, R. (2024). The SAFE procedure: a practical stopping heuristic for active learning-based screening in systematic reviews and meta-analyses. Systematic Reviews, 13(1). https://doi.org/10.1186/s13643-024-02502-7

**Comment by Rensvandeschoot** [comment]
Hi! Great to see you're planning ahead and using ASReview thoughtfully. Thanks for sharing your preregistration and SAFE-based strategy.

Since ASReview v2, we’ve made several important improvements, especially around model performance and usability. Based on the new version, we would suggest updating your screening procedure as follows:

**Phase 1 – Calibration instead of random training**
Rather than screening a minimum percentage randomly, we now recommend a small calibration set to align the team on the inclusion criteria. Random screening of 1–10% is no longer advised, especially when working with large, unfiltered datasets — this would be inefficient and potentially unethical. All you need are a few relevant papers [to start](https://asreview.readthedocs.io/en/latest/lab/screening.html). The software will first show you a random paper, which is most likely irrelevant, and then it starts training its model. 

**Phase 2 – Use the default U4 model, skip RR_T**
Please consider switching from the Naive Bayes model to the new default U4 model, which offers much better performance (details: [ASReview models](https://asreview.readthedocs.io/en/latest/lab/models.html)).
We also recommend not using the RR_T formula or fixed percentages (e.g., 10%) as stopping rules. These don’t scale well with large datasets and may lead to unnecessary screening. Instead, ensure key papers are found and consider using the recall plot and other [stopping suggestions](https://asreview.readthedocs.io/en/latest/lab/progress.html#stop-screening) provided in the dashboard.

**Phase 3 – Model switching is now possible in v2**
You can now switch to a different model within the same project. The heavy models require installing the [Dory package](https://asreview.readthedocs.io/en/latest/lab/dory.html), which will install all kinds of heavy machine learning dependencies. Just be aware this has hardware requirements (see: [Heavy models](https://asreview.readthedocs.io/en/latest/lab/models.html#elas-heavy)) and after the optimization of the hyperparameters in version 2, I am not sure if switching is still absolutely necessary.

**Phase 4 – Still relevant, but simpler**
You can still assess model quality with screening the "irrelevant" records from the previous phase, but training with just the relevant ones using the U4 model is often enough in v2. You no longer need to manually select 10 high/low-ranked records. This will save you time!

**Full-text screening – New options available**
You can combine abstract and full-text screening more flexibly now, using the DOI link and the new tagging feature ([docs](https://asreview.readthedocs.io/en/latest/lab/project_create.html#add-tags)). This way, you can use a tag for the abstract inclusion phase, but use the labeling decision for full-text inclusion. This way, you can train the AI on the most important papers only. It requires you to spend more time on one record, but it will be worth it because it is even possible to add all kinds of labels for data extraction, and you will finish the entire process more quickly!!

Hope this helps to align your preregistration with the latest ASReview version! Let us know if you’d like help revising your protocol based on these suggestions.

**Reply by AnneliesPieterman-Bos** [reply]
Thanks so much for the feedback, @Rensvandeschoot!

A couple of follow-up questions:

- phase 1: do you mean that I load the full dataset into ASReview and then we start screening as an author team without selecting prior knowledge to collaboratively decide for the first randomly selected title-abstracts whether to include them. And then the model starts training on these data? And then we stop when we feel we are aligned on the inclusion criteria?
- phase 4: how can an independent screener check a random set of title-abstracts that the original screeners marked as irrelevant in phase 1-3 (i.e., using validation mode) in version 2.0? I don't see a validation mode anymore.

**Reply by Rensvandeschoot** [reply]
Great follow-up questions!

### **Phase 1: Calibration phase options**

Yes, there are several valid ways to do Phase 1, depending on how you want to collaborate:

* **Option 1**: Ask all team members to select a few records they *think* are relevant. Then discuss them together to refine the inclusion criteria.
* **Option 2**: Select a mix of older and newer records to see if your criteria work consistently over time.
* **Option 3**: Take a completely random sample from the dataset and screen that together.

You *can* use ASReview for this, but it's not required. In other projects, we’ve used slides in a team meeting, a shared Excel file, or, indeed, ASReview with only a subset of records loaded. 
If you want to use ASReview for Phase 1 screening only, you can **import just the calibration set** to benefit from the ASReview interface, without needing the full dataset.

Alternatively, if you prefer to import the full dataset from the start, you can choose a **custom model with a “random” query strategy** to simulate random screening for Phase 1. Just remember: this won’t train a useful model, but it allows you to discuss inclusion decisions based on random records.

Once your team feels confident and aligned, you stop Phase 1 and move on to active learning in Phase 2.


### **Phase 4: Re-checking excluded records**

You're right that **there is no validation mode in ASReview v2** like there was in earlier versions, but if you import a fully labelled dataset in the screening mode, you will see an indicator with the initial label, see the [docs](https://asreview.readthedocs.io/en/latest/lab/validation.html).

But in Phase 4 (as defined in the SAFE procedure), the goal isn’t to sample randomly — it’s to **recheck the most likely missed relevant papers** among those previously marked as irrelevant.

So instead of using a random subset, you:

* Import the **excluded records** (from Phases 2 and 3)
* Use active learning again to rank these based on what the model learned from previously *included* records
* Have an independent screener check the top-ranked records to assess potential false negatives

This step acts as a quality check — not for calculating IRR, but for catching any possibly overlooked relevant studies.

**Reply by Rensvandeschoot** [reply]
In this paper we describe the new Elase-models and the performance increase: [ASReview LAB v.2: Open-source text screening with multiple agents and a crowd of experts](https://doi.org/10.1016/j.patter.2025.101318)
